### DISCUSSION TITLE
Need help with x-bind:href

### DISCUSSION THREAD

**Discussion by huylv** [alpinejs/alpine#4162] [1. Help] [answered]
`x-bind:href` does not seem to work for me? I have a super simple `<a/>` element as follows and it is not displayed as a hyperlink (not even clickable) somehow:

    <a x-bind:href="https://example.com">Click me!</a>

Am I doing something wrong?

**Comment by ekwoka** [comment]
Is that the full code?

Does a parent element have an x-data attribute?

**Reply by huylv** [reply]
Here is the full HTML:

    <!DOCTYPE html>
    <html lang="en">
    <head>
      <meta charset="UTF-8">
      <script src="//unpkg.com/alpinejs" defer></script>
    </head>
    <body>
      <div x-data="{ url: 'https://example.com' }">
          <a x-bind:href="url">Click me!</a>
      </div>
    </body>
    </html>

**Reply by ekwoka** [reply]
https://awesomealpine.com/play/?coreplugins=0&html=PGRpdiB4LWRhdGE9InsgdXJsOiAnaHR0cHM6Ly9leGFtcGxlLmNvbScgfSI-CiAgPGEgeC1iaW5kOmhyZWY9InVybCI-Q2xpY2sgbWUhPC9hPgo8L2Rpdj4K&script=Y29uc29sZS5sb2coJ2hlbGxvJyk7Cg

That totallly works, so I'm not sure what issue you're having.

**Reply by huylv** [reply]
I put your code in an HTML file, this is how it is shown:
![image](https://github.com/alpinejs/alpine/assets/2832672/9550a743-cb7c-4d43-a6d8-7c05df141c3e)

As you can see, the text in the browser is just plain text, not clickable at all. I tried on Chrome, Safari on Linux and macOS, same result.

**Reply by huylv** [reply, marked as answer]
Oh, never mind. I change the `<script/>` tag to
    
    <script defer src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js"></script>

and it works. Sorry, I really appreciate your help.

**Reply by SimoTod** [reply]
FYI you can't use `//` when developing from a file, as it generates a protocol relative URL. You need to explicitly put the protocol `https://unpkg.com/alpinejs"` but nevertheless `cdn.jsdelivr.net` is a better CDN than `unpkg.com`
